debug.log

```diff
@@ -0,0 +1 @@
+[0118/133135.552:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)
```

Comment:
I don't think we need this file checked in :)

Reply:
Oh. Which file?

Reply:
The one that I commented on, `debug.log`.

Reply:
debug.log. Got it. Why was I so confused about this? lol